### DISCUSSION TITLE
ninja: error: manifest 'build.ninja' still dirty after 100 tries on android

### DISCUSSION THREAD

**Discussion by milon27** [software-mansion/react-native-reanimated#5339] [Q&A]
RN 0.72.3 + react-native-reanimated 3.4.1 Android

if I sync the project on android studio and run the App it works, but if I try to rebuild the project on android studio or
run `react-native run-android` got this error

note: I am using mono repo following this guide: https://medium.com/@shuklaaman892/mono-repo-react-18-2-0-and-react-native-0-72-3-project-using-yarn-workspace-a85dc443707b. 

![image](https://github.com/software-mansion/react-native-reanimated/assets/44096479/2ed5df7a-d833-4a81-b281-3f1f7d7b3702)

**Comment by sourdragon** [comment]
Getting the same issue on react native 73.6 . But i dont think i can reproduce it .

**Comment by muhammadabubakarrana** [comment]
I am also getting the same issue

**Comment by MatiPl01** [comment]
Do you build the project on Windows?
If you do, it might be the path length limit issue. I found very similar issue reports on different library repo and suggestions to move the project to another location seem to be the valid solution.
Can you move the project to another path as suggested in [this](https://github.com/mrousavy/react-native-vision-camera/issues/1941#issuecomment-1757159751) comment and let me know if you still experience the same issue?

**Reply by MatiPl01** [reply]
Alternatively, you can try to install `ninja` manually and provide it as suggested in [this](https://github.com/ninja-build/ninja/issues/1900#issuecomment-1828258152) comment.
